snuba/datasets/cdc/groupassignee.py

```diff
@@ -82,20 +83,17 @@ def __init__(self) -> None:
             query_processors=[PrewhereProcessor()],
         )
 
-        storage_selector = SingleStorageSelector(storage=storage)
-
         super().__init__(
             storages=[storage],
-            storage_selector=storage_selector,
+            query_plan_builder=SingleTableQueryPlanBuilder(
+                storage=storage, post_processors=[],
+            ),
```

Comment:
Should `post_processors` have a default value if it commonly will be an empty list? Or is there value in explicitly calling out the lack of post-processors?